### ISSUE TITLE
setup-jfrog-cli should be added with latest JFrog CLI as default version

### ISSUE THREAD

**Issue by amithkm** [jfrog/setup-jfrog-cli#36]
**Is your feature request related to a problem? Please describe.**
If we want to enjoy the latest fixes and new features and be update to date with JFrog CLI version, we need to manually add the version in the workflow.
**Describe the solution you'd like to see**
It would be great if setup-jfrog-cli will have the default JFrog CLI version as the latest version.
**Describe alternatives you've considered**
We can achieve this by modifying the workflow with the latest version of JFrog CLI manually. 
uses: jfrog/setup-jfrog-cli@v1
  with:
    version: X.Y.Z
**Additional context**

**Comment by callum-tait-pbx** [comment]
Looks like this is a thing already? https://github.com/jfrog/setup-jfrog-cli/releases

**Comment by yahavi** [comment]
We can add this functionality as an option. i.e:
```yaml
- uses: jfrog/setup-jfrog-cli@v1
  with:
    version: latest
```

**Comment by callum-tait-pbx** [comment]
Just for my knowledge @yahavi how does that differ from just referencing the v1 tag? I can see you already provide a v1 tag which seems to get updated with whatever the latest v1 release is?

**Comment by yahavi** [comment]
@callum-tait-pbx,
The v1 tag is this Action tag - not the JFrog CLI tag. It means - use the latest release of "Setup JFrog CLI" under v1 major (currently v1.1.4). We put our best efforts to make this version as stable as possible.

This CLI version in v1 is not always the latest. The v1 tag contains a JFrog CLI version that was thoroughly tested for compatibility.

**Comment by davereid-pfg** [comment]
@yahavi Is with version: latest supported as per your comment or do we need to build out support for that? I got very hung up that we encountered some bugs that have been fixed in the latest version, while this uses 2.3.0 by default.

**Comment by eyalbe4** [comment]
Version 2.1.0 of the **setup-jfrog-cli** GitHub Action is now released, and includes support for downloading the latest JFrog CLI release.

![image](https://user-images.githubusercontent.com/7830056/151979900-15a0f88f-6a9d-4a7a-af5d-95d1a587f57e.png)
